src/views/TechnicalAssistance/AdminHome/components/ReviewAdviceLetter/RecommendationsList.tsx

```diff
@@ -90,7 +91,8 @@ export default function RecommendationsList({
                 <h4 className="margin-bottom-1">{title}</h4>
               )}
 
-              <p className="margin-y-1">{description}</p>
+              {/* <p className="margin-y-1">{description}</p> */}
+              <RichTextViewer className="margin-y-1" value={description} />
```

Comment:
Should there be an import & use of the external link modal hook here? I think having to remember to import `useRichTextViewerLinkModal` and use it in each place we reference `RichTextViewer` might leave room for human error

I think it might make the most sense to reference the External Link Modal _inside_ the RichTextViewer, that way any instances of the viewer automatically handle links without the calling component needing to import the hook - @adamodd any thoughts on this?

Reply:
> Should there be an import & use of the external link modal hook here? I think having to remember to import useRichTextViewerLinkModal and use it in each place we reference RichTextViewer might leave room for human error

RecommendationsList looks like it's only used in the ReviewAdviceLetter parent so the hook there is sufficient.

@ClayBenson94 do you want to include a Modal inside RT Viewer so that there is 1 modal for 1 viewer?

Reply:
There are some tradeoffs on the diff approaches to connect Viewer Links to the modal.

Though this approach might be more prone to misuse, which I think isn't too bad, it's simpler to place in and you don't have to handle references between components and don't need to modify much surrounding code.

Reply:
My initial thoughts were that having the 1:1 relationship between Viewer <> Modal was useful, since there'd be no need to understand how the hook used in one component affects another (as mentioned - RecommendationsList is used inside ReviewAdviceLetter, and the hook used there affects the viewer in this component, which could be a bit hard to understand)

I would expect that including the Modal in the Viewer would be both less prone to misuse, but also simpler, as any new instances of the viewer can just import the viewer & pass it the data, rather than having to connect multiple components and hooks in each place we use it.

Reply:
I think implementation-wise it's simpler to bundle the hook functionality with the viewer.  This removes the burden on the user to remember to hook up something that is a security concern.  Also if we want in the future, we could pass a prop  to the view _if_ we ever wanted to disable the modal functionality.

Reply:
If you want @adamodd I'd be happy to sync so we can brainstorm the best approach

Reply:
There's nothing really complicated about this, it looks like there was just different impressions about approach.

@ClayBenson94 @patrickseguraoddball the modal is included in the viewer now. Let me know if this works for you.